internal/watcher/watcher_test.go

```diff
@@ -341,6 +341,60 @@ func TestUpdateDirsFailing(t *testing.T) {
 	assertGPTVersionEquals(t, destRemove, 3)
 }
 
+func TestUpdateDirsWithEmptyDirSlice(t *testing.T) {
```

Comment:
we couldn’t have it added to the previous table testing with a WantErr?

Reply:
ack, let’s have a look in a separate PR if this could be refactored. If you feel there is not enough commonality, feel free to discare.